probe/Dockerfile

```
@@ -0,0 +1,9 @@
+FROM python:3.11-slim:latest
```

Hadolint (2.14.0)

[error] unexpected ':'
expecting '@', '\', a new line followed by the next instruction, or the image tag

(DL1000)